src/storage/internal.go

```diff
@@ -410,3 +411,47 @@ func (insto InternalStorage) GetAssociatedClasses(rule string) ([]Class, error)
 	rows.Close()
 	return classes, err
 }
+func (into InternalStorage) DeleteClasses(ids []int64) (bool, error) {
```

Comment:
add a variable with initial value 0 to check the number of affected rows. 

return true if it is greater than 0